test/smart-wallet/gasEstimations.ts

```diff
@@ -179,8 +181,8 @@ describe("Wallet tx gas estimations with and without refunds", function () {
     const walletOwner = await userSCW.owner();
     expect(walletOwner).to.equal(owner);
 
-    const walletNonce1 = await userSCW.getNonce(0); // only 0 space is in the context now
-    const walletNonce2 = await userSCW.getNonce(1);
+    const walletNonce1 = await userSCW.getNonce(ACCOUNT_ABSTRACTION_FLOW);
```

Comment:
maybe update the test in verifying-paymaster-test.ts to use 0 nonce space when creating and signing actual userop.